docs/src/extend/index.md

```diff
@@ -21,34 +21,26 @@ In order to work with ESLint as a developer, it's recommended that:
 
 If that sounds like you, then continue reading to get started.
```

Comment:
I think the opening section needs to be rewritten because this page is now only about extending ESLint while the current text in this section is mostly about contributing to ESLint.

Reply:
addressed in aaad16323